docs/style.md

```diff
@@ -0,0 +1,50 @@
+---
+id: style
+title: Styleguides
+sidebar_label: Styleguides
+---
+
+---
+
+
+On `amphora-html` using the `data`  and `meta` object that are the instances of the components and site respectively. This state
+will expose the component variations on the state then it will set these variations on the into a structure [Kiln](https://docs.clayplatform.com/clay-kiln/) will need then
```

Comment:
```suggestion
will expose the component variations on the state, then it will set these variations into a structure. Finally, [Kiln](https://docs.clayplatform.com/clay-kiln/) will render the `HTML`
```